qiskit/circuit/parameterexpression.py

```diff
@@ -90,12 +90,16 @@ def assign(self, parameter, value: ParameterValueType) -> "ParameterExpression":
             return self.subs({parameter: value})
         return self.bind({parameter: value})
 
-    def bind(self, parameter_values: Dict) -> "ParameterExpression":
+    def bind(
+        self, parameter_values: Dict, allow_unknown_parameters: bool = False
+    ) -> "ParameterExpression":
         """Binds the provided set of parameters to their corresponding values.
 
         Args:
             parameter_values: Mapping of Parameter instances to the numeric value to which
                               they will be bound.
+            allow_unknown_parameters: If False, raises an error if parameter_values
+                contains Parameters outside those in self
```

Comment:
Would be nice to format this for the docs 🙂 
```suggestion
            allow_unknown_parameters: If ``False``, raises an error if ``parameter_values``
                contains Parameters in the keys outside those present in the expression.
```

Reply:
Done, thanks.